packages/commerce-sdk-react/README.md

```diff
@@ -1,24 +1,51 @@
 # Commerce SDK React
 
-A library of React hooks for fetching data from Salesforce B2C Commerce.
+<p align="center">
+A collection of React hooks for <b>fetching</b>, <b>caching</b>, and <b>mutating data</b> from the <b><a href="https://developer.salesforce.com/docs/commerce/commerce-api/overview">Salesforce B2C Commerce API</a></b> (SCAPI).
+The library contains declarative, always-up-to-date auto-managed <b>queries</b> and <b>mutations</b> for a more convenient developer experience.
```

Comment:
Here are my thoughts on this section:

1. It's very market-y, and I don't really believe it reflects what the library actually is. We call it a "collection of React hooks" but I think we have previously settled that it's a collection of ReactQuery hooks which is more accurate.
2. Another part of our code base is the managing of authentication which we don't mention here.
3. We are also using a lot of ReactQuery terms like mutations/caching/queries as though it was something that we created. Instead I'd mentioned that we are taking advantage of these features that ReactQuery provides us.
4. "always-up-to-date auto-managed queries and mutations" I don't really know what that means. I think you might be referencing that is helps with upgradability with templates? which would be out of context.

Reply:
good feedback! Removed this confusing sentence and explained the tight integration with react query later in this doc